src/rules/preferObjectSpreadRule.ts

```diff
@@ -54,6 +54,17 @@ function walk(ctx: Lint.WalkContext<void>) {
             // Object.assign(...someArray) cannot be written as object spread
             !node.arguments.some(isSpreadElement)) {
             if (node.arguments[0].kind === ts.SyntaxKind.ObjectLiteralExpression) {
+
+                /**
+                 * @url https://github.com/palantir/tslint/issues/3117
+                 * Per TS2698, "spread types may only be created from
+                 * object types."
```

Comment:
also add a TODO to remove this when typescript get's support for spread types and a link to the PR (linked in the issue comments)